### PR TITLE
backports/py-datadog: new aport

### PR DESCRIPTION
**Pull request by seiga-k**

### CODE REVIEW

File: backports/py-datadog/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source="$pkgname-$pkgver.tar.gz::https://github.com/DataDog/$_pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python2 setup.py build
+	python3 setup.py build
+}
+
+package() {
+	depends="py2-${pkgname#py-} py3-${pkgname#py-}"
```
Comment (at-wat) [inline backports/py-datadog/APKBUILD]: this `depends` seems wrong.

File: backports/py-datadog/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	install -D -m 644 LICENSE \
+		"$pkgdir"/usr/share/licenses/$pkgname/LICENSE
+}
+
+_py() {
```
Comment (at-wat) [inline backports/py-datadog/APKBUILD]: https://github.com/seqsense/aports-ros-experimental/blob/dc9690604a4b95e4478142ea2d860df5c0362c03/backports/py-rosinstall/APKBUILD#L45-L47
`install_if` should be added and updating `pkgdesc` is recommended.

**Comment by seiga-k** [comment]
I completely miss understood about dependency. I'll check and try.

**Comment by at-wat** [comment]
pytest.monkeypatch.context is required in the test.
requires pytest>=3.6.0
https://github.com/pytest-dev/pytest/commit/659c044372ece204146fee9164fb6144c494eb58

**Comment by at-wat** [comment]
```
E           AttributeError: 'str' object has no attribute 'decode'
/usr/lib/python3.8/site-packages/pkg_resources/__init__.py:1424: AttributeError
```

**Comment by at-wat** [comment]
missing requirements of pytest backport
```
ERROR: unsatisfiable constraints:
  .makedepends-pytest-0:
    masked in: cache
    satisfies: world[.makedepends-pytest]
  py2-atomicwrites (missing):
    required by:
  py2-more-itertools (missing):
    required by:
  py2-pluggy (missing):
    required by:
  py3-atomicwrites (missing):
    required by:
  py3-more-itertools (missing):
    required by:
  py3-pluggy (missing):
    required by:
```

**Comment by at-wat** [comment]
these failing tests also fails on local run

**Comment by at-wat** [comment]
@seiga-k all dependencies are backported

**Comment by at-wat** [comment]
@seiga-k ping, this should be ready

**Comment by at-wat** [comment]
@seiga-k ping

**Comment by seiga-k** [comment]
Sorry for late. I could not work on this topic these days.

In my investigation, these packages are needed to clear all dependencies. 

https://github.com/micheles/decorator
https://github.com/psf/requests
https://github.com/chardet/chardet
https://github.com/kjd/idna
https://github.com/urllib3/urllib3
https://github.com/google/brotli
https://github.com/TypeError/secure.py
https://github.com/pyca/pyopenssl
https://github.com/pyca/cryptography
https://github.com/benjaminp/six
https://github.com/certifi/python-certifi
https://github.com/phihag/ipaddress
https://github.com/Anorov/PySocks

Some packages are listed on official alpine sports but it's only for python3. So I thought it needs python2 APKBUILD, isn't it correct?

**Comment by at-wat** [comment]
I have already ported all [dependencies declared by datadogpy/setup.py](https://github.com/DataDog/datadogpy/blob/1cbb1bbe204adc3306f6d9aa95fb51b6f54db43b/setup.py#L18-L27) in this branch.

**Comment by seiga-k** [comment]
Yes, I read it and then I thought ```decorator``` and ```requests``` need to install. and those packages are not included default alpine aports.
Isn't it correct?

**Comment by at-wat** [comment]
`decorator` and `requests` are quite common packages and provided by Alpine official repository.
https://pkgs.alpinelinux.org/packages?name=py2-decorator&branch=v3.7&arch=x86_64
https://pkgs.alpinelinux.org/packages?name=py2-requests&branch=v3.7&arch=x86_64

**Comment by seiga-k** [comment]
Ah, OK, I mistake an alpine version. Thank you.

**Comment by seiga-k** [comment]
Should I merge this PR? or you will?

**Comment by at-wat** [comment]
I'll update commit message and merge.

**Pull request merged**